dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/engine/command/impl/CuratePublishedDatasetVersionCommand.java

```diff
@@ -113,6 +113,9 @@ public Dataset execute(CommandContext ctxt) {
         updateVersion.setLastUpdateTime(getTimestamp());
         tempDataset.setModificationTime(getTimestamp());
 
+        // Mark dataset for reharvest
+        tempDataset.setLastChangeForExporterTime(getTimestamp());
```

Comment:
Maybe just private method called 'markForReharvest'?

Reply:
I wanted to be consistent with the rest of this class but sure whatever ;)